buildprocess/generateDocs.ts

```diff
@@ -20,7 +20,7 @@ import Trait from "../lib/Traits/Trait";
 
 /** Get type name for a given Trait */
 function markdownFromTraitType(trait: Trait) {
-  let base = "";
+  let base;
```

Comment:
If removing the default string value, can you add an explicit type definition so it is not typed as any

Reply:
Fixed.